### PR TITLE
removed unnecessary cd_user

### PR DESCRIPTION
**Pull request by tjaeschke**

### CODE REVIEW

File: src/org/ods/OdsContext.groovy [hunk 1]
```diff
@@ -385,12 +385,7 @@ class OdsContext implements Context {
       returnStdout: true, script: 'git config --get remote.origin.url',
       label : 'getting GIT url'
     ).trim()
-    // check if the url already contains a user
-    if (!gitUrl.contains('@')) {
-      return gitUrl.replace('://', '://cd_user@')
-    } else {
-      return gitUrl
-    }
+    return gitUrl
```
Comment (michaelsauter) [inline src/org/ods/OdsContext.groovy]: Are you 100% sure? I am worried that we did this replacement for a reason 😆

Comment (tjaeschke) [reply src/org/ods/OdsContext.groovy]: I would suggest, we keep all project specifix cd user PRs open until end of week or thursday evening.

Comment (michaelsauter) [reply src/org/ods/OdsContext.groovy]: OK. I tried to trace usages of this variable and I would agree that theoretically it should be fine. The image label will change (in `stageStartOpenshiftBuild`), but that should be OK.

**Comment by tjaeschke** [comment]
relates to https://github.com/opendevstack/ods-project-quickstarters/issues/289

**Pull request merged**